migration/identity-migration/src/main/java/io/camunda/migration/identity/config/saas/StaticEntities.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.migration.identity.config.saas;
+
+import io.camunda.service.AuthorizationServices.CreateAuthorizationRequest;
+import io.camunda.service.RoleServices.CreateRoleRequest;
+import io.camunda.zeebe.protocol.record.value.AuthorizationOwnerType;
+import io.camunda.zeebe.protocol.record.value.AuthorizationResourceType;
+import io.camunda.zeebe.protocol.record.value.PermissionType;
+import java.util.List;
+import java.util.Set;
+
+public class StaticEntities {
```

Comment:
💭 Not sure about the name of this class, any idea ?

Reply:
I'd prefer to have them in a separate class because I think otherwise the handlers might be too chaotic, instead they should just handle what they have to handle.
IMO having this constant in a separate class is more clear and readable then put everything in the handler